src/main/java/com/mixpanel/android/mpmetrics/AutomaticEvents.java

```diff
@@ -0,0 +1,14 @@
+package com.mixpanel.android.mpmetrics;
+
+/* package */ public class AutomaticEvents {
+    public static final String FIRST_OPEN = "$ae_first_open";
+    public static final String FIRST_OPEN_DATE = "$ae_first_app_open_date";
+    public static final String SESSION = "$ae_session";
+    public static final String SESSION_LENGTH = "$ae_session_length";
+    public static final String TOTAL_SESSIONS = "$ae_total_app_sessions";
+    public static final String TOTAL_SESSIONS_LENGTH = "$ae_total_app_session_length";
+    public static final String APP_UPDATED = "$ae_updated";
+    public static final String VERSION_UPDATED = "$ae_updated_version";
+    public static final String APP_CRASHED = "$ae_crashed";
+    public static final String APP_CRASHED_REASON = "$ae_crashed_reason";
```

Comment:
i can see why it's useful in js but can't see the use case in java since you can follow references. i prefer not to bring this methodology in the sdk :)